pkg/controller/cluster/cluster.go

```diff
@@ -178,6 +179,11 @@ func (c *ClusterReconciler) Reconcile(ctx context.Context, req reconcile.Request
 }
 
 func (c *ClusterReconciler) createCluster(ctx context.Context, cluster *v1alpha1.Cluster) error {
+
+	if cluster.Name == clusterInvalidName {
+		klog.Errorf("Invalid cluster name %s, no action will be taken", cluster.Name)
```

Comment:
It's fine to return nil here?

Reply:
yes I don't want the controller to reconcile again, just show an error and stop processing this cluster object, in the future, we need to add the status of the cluster in the cluster object as well